pkgs/development/python-modules/blist/default.nix

```diff
@@ -14,6 +15,14 @@ buildPythonPackage rec {
     sha256 = "1hqz9pqbwx0czvq9bjdqjqh5bwfksva1is0anfazig81n18c84is";
   };
 
+  patches = [
+    # python 3.7 compatibility
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/DanielStutzbach/blist/pull/78.diff";
```

Comment:
these kind of diffs are typically not stable. Please take an actual commit. Otherwise, squash them together and include the patch as a file here.

Reply:
resolved